packages/lib-classifier/src/components/Classifier/ClassifierContainer.js

```diff
@@ -159,7 +173,7 @@ export default function ClassifierContainer({
   }, [loaded, authClient])
 
   try {
-    if (loaded) {
+    if (loaded && workflowSnapshot?.id) {
```

Comment:
`workflowID` is a required prop in the classifier so I think it should be checked here. 

I've tried to keep track of the last workflow ID that was passed down from the URL, to fix a bug where the classifier renders with an undefined workflow ID if you go to `/classify`.